src/model.ts

```diff
@@ -89,7 +145,29 @@ export class HubApiModel {
   }
 
   // TODO remove when koop-core no longer requires
-  getData () {}
+  getData() { }
+
+
+  private async fetchHubSiteModel(hostname, opts): Promise<IModel> {
+    try {
+      return await fetchSiteModel(hostname, opts);
+    } catch (err) {
+      // Throw 404 if domain does not exist (first) or site is private (second)
+      if (err.message.includes(':: 404') || err.response?.error?.code === 403) {
```

Comment:
Confirm with @thomas-hervey that we want to mask 403s as 404s

Reply:
@rgwozdz @sansth1010 is this a new conditional? IIRC this it the behavior today. I'm happy to hear your input on this or status code best practices. Generally I think it feels odd to mask status codes, but we wanted to prevent users from _knowing_ that a feed was available on a private Hub site, not just prevent them from accessing it.

Reply:
I've seen the same pattern in discussions, generally don't like it.  Why can't they know about it?

Reply:
Found out that we can get the information we need without needing `siteModel` for data enrichment. Therefore, removed it entirely from the output plugin.